Happy Dance/Assets/_Scripts/EnableGirl.cs

```diff
@@ -15,28 +15,34 @@ public class EnableGirl : MonoBehaviour
 
         private void Start()
         {
-
-        objectToActivate.SetActive(false);
-        StartCoroutine(ActivationRoutine());
-
-		//audioSource = mainCamera.Find.GetComponent<AudioSource>();
-
-		//AudioSource.PlayClipAtPoint(beam, new Vector3(5, 1, 2));
-
+            if (objectToActivate != null)
+            {
+                objectToActivate.SetActive(false);
+            }
+        
+            StartCoroutine(ActivationRoutine());
         }
 
         public IEnumerator ActivationRoutine()
         {
             //Wait for 14 secs.
             yield return new WaitForSeconds(timer);
 
+            if (objectToActivate != null)
+            {
+                objectToActivate.SetActive(true);
+            }
+
             //Turn My game object that is set to false(off) to True(on).
-            objectToActivate.SetActive(true);
+            
```

Comment:
remove the comments.

Reply:
That's not my script. I only added a null check to prevent issues when the experience is built.